RHIDP-3974 managing authorization using the Web UI (#633)
redhat-developer/red-hat-developers-documentation-rhdh: assemblies/assembly-managing-authorizations-by-using-the-rhdh-web-ui.adoc

[id='proc-rbac-ui-manage-roles_{context}']
= Managing role-based access controls (RBAC) using the {product} Web UI

Policy administrators can use the {product-short} web interface (Web UI) to allocate specific roles and permissions to individual users or groups. Allocating roles ensures that access to resources and functionalities is regulated across the {product-short}.

> With the policy administrator role in {product-short}, you can assign permissions to users and groups, which allow users or groups to view, create, modify, and delete the roles using the {product-short} Web UI.

Reviewer: Looks great, only have 1 suggestion for you:

```suggestion
With the policy administrator role in {product-short}, you can assign permissions to users and groups. This role allows you to view, create, modify, and delete the roles using  {product-short} Web UI.
```
